packages/connector/src/services/core/test/controllers/ildcp-controller.test.ts

```diff
@@ -61,7 +61,7 @@ describe('ILDCP Controller', function () {
     )
   })
 
-  test('returns an ildcp response if incoming account is not a peer', async () => {
+  test.skip('returns an ildcp response if incoming account is not a peer', async () => {
```

Comment:
Why skip?

Reply:
The ILDCP controller currently doesn't distinguish a peer account. It just always gets an address from `ctx.services.accounts.getAddress`.
https://github.com/coilhq/rafiki/pull/18/files/f60e5412527fe7db202f2c87bd39661fad31cddf#diff-07bf7f9a9ab11803e477e436f9379f39032dd932b4225464ea0f198bd8d096a9L20-R21
Should it compare with the server's ILP address?

Reply:
Ah, you're right; skipping is fine.